### DISCUSSION TITLE
Motif enrichment score in individual cells

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [GreenleafLab/ArchR#1063] [Questions / Documentation]
Hi, where/how can I extract enrichment score for each TF in individual cells? I am trying to compare the motif enrichment score between two known clusters in the differential peaks. I am only able to find whether a motif is enriched or not in a given peak.

**Comment by rcorces** [comment]
https://www.archrproject.com/reference/peakAnnoEnrichment.html

**Reply by Chhiring-Lama** [reply]
Thank you for the quick reply @rcorces! I can only the average enrichment score for the motifs. Just like how get deviation and zscore for the motifs in each cells from the Motifmatrix, is there a way retrieve the enrichment score in each cells?

**Reply by rcorces** [reply]
Sorry for not understanding your initial question. No - you cannot obtain per-cell enrichment since you dont have per-cell peak calls. What you're looking for is effectively provided by chromVAR

**Reply by Chhiring-Lama** [reply]
I will try using chromVAR. Thanks!
